unit-tests/func/alt-ir/test-streamimg-before.cpp

```diff
@@ -0,0 +1,34 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+
+#include "../../test.h"
+#include "../func-common.h" //todo functional-common.h
+#include "alt-ir-common.h" //todo alt-ir-common.h
+
+
+using namespace rs2;
+
+TEST_CASE( "AC fails if AltIR was enabled before stream start", "[l500][live]" )
```

Comment:
I thought this is covered in another file...

Reply:
done